ps_checkout.php

```diff
@@ -133,7 +137,7 @@ public function __construct()
 
         // We cannot use the const VERSION because the const is not computed by addons marketplace
         // when the zip is uploaded
-        $this->version = '7.3.6.3';
+        $this->version = '7.4.2.2';
```

Comment:
It can be 7.4.2.0, actually, as you prefer.

Reply:
And here